xtask/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "xtask"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+anyhow = "1.0.71"
+# cargo-run-wasm = "0.3.2"
+cargo-run-wasm = { version = "0.3.2", git = "https://github.com/ErichDonGubler/cargo-run-wasm", branch = "expose-args" }
```

Comment:
- [ ] This needs to be resolved upstream before merging, _or_ we need to commit to using my Git branch for now. I don't particularly care which we do right now.

Reply:
We can accept a git dep here, it's fine

Reply:
Okay! 😁